org.mwc.cmap.legacy/build.xml

```diff
@@ -0,0 +1,202 @@
+		<mkdir dir="${temp.folder}"/>
+		<antcall target="build.jars"/>
+		<antcall target="gather.bin.parts">
+			<param name="destination.temp.folder" value="${temp.folder}/"/>
+		</antcall>
+		<jar destfile="${plugin.destination}/org.mwc.cmap.legacy_1.0.255.jar" basedir="${temp.folder}/org.mwc.cmap.legacy_1.0.255" filesetmanifest="merge"/>
+		<delete dir="${temp.folder}"/>
+	</target>
+
+	<target name="cmap_legacy.jar.nestedJars">
+	</target>
+	<target name="cmap_legacy.jar" depends="init,cmap_legacy.jar.nestedJars" unless="cmap_legacy.jar" description="Create jar: org.mwc.cmap.legacy cmap_legacy.jar.">
+		<delete dir="${temp.folder}/cmap_legacy.jar.bin"/>
+		<mkdir dir="${temp.folder}/cmap_legacy.jar.bin"/>
+		<path id="cmap_legacy.jar.classpath">
+			<pathelement path="../org.mwc.cmap.jfreechart/libs/jcommon-1.0.23.jar"/>
+			<pathelement path="../org.mwc.cmap.jfreechart/libs/jfreechart-1.0.19-experimental.jar"/>
+			<pathelement path="../org.mwc.cmap.jfreechart/libs/jfreechart-1.0.19-swt.jar"/>
+			<pathelement path="../org.mwc.cmap.jfreechart/libs/jfreechart-1.0.19.jar"/>
+			<pathelement path="libs/junit-4.12.jar"/>
+			<pathelement path="libs/hamcrest-core-1.2.jar"/>
+			<pathelement path="libs/openmap.jar"/>
+			<pathelement path="libs/jmf.jar"/>
+			<pathelement path="libs/vrml97.jar"/>
+			<pathelement path="libs/flanagan.jar"/>
+			<pathelement path="libs/jackson-core-asl-1.9.4.jar"/>
+			<pathelement path="libs/jackson-mapper-asl-1.9.4.jar"/>
+			<pathelement path="libs/joda-time-2.1.jar"/>
+			<pathelement path="libs/javalite-common-1.2-20120312.200900-220.jar"/>
+			<pathelement path="libs/commons-math3-3.3.jar"/>
+		</path>
+		<!-- compile the source code -->
+		<javac destdir="${temp.folder}/cmap_legacy.jar.bin" failonerror="${javacFailOnError}" verbose="${javacVerbose}" debug="${javacDebugInfo}" includeAntRuntime="no" bootclasspath="${bundleBootClasspath}" source="${bundleJavacSource}" target="${bundleJavacTarget}"		>
+			<compilerarg line="${compilerArg}" compiler="${build.compiler}"/>
+			<classpath refid="cmap_legacy.jar.classpath" />
+			<src path="src/"			/>
+			<compilerarg value="@${basedir}/javaCompiler.cmap_legacy.jar.args" compiler="org.eclipse.jdt.core.JDTCompilerAdapter"			/>
+			<compilerarg line="-log &apos;${temp.folder}/cmap_legacy.jar.bin${logExtension}&apos;" compiler="org.eclipse.jdt.core.JDTCompilerAdapter"			/>
+		</javac>
+		<antcall target="checkCompilationResults"/>
+		<!-- Copy necessary resources -->
+		<copy todir="${temp.folder}/cmap_legacy.jar.bin" failonerror="true" overwrite="false">
+			<fileset dir="src/">
+				<exclude name="**/*.java"/>
+				<exclude name="**/package.htm*"/>
+			</fileset>
+		</copy>
+		<mkdir dir="${build.result.folder}"/>
+		<jar destfile="${build.result.folder}/cmap_legacy.jar" basedir="${temp.folder}/cmap_legacy.jar.bin"/>
+		<delete dir="${temp.folder}/cmap_legacy.jar.bin"/>
+	</target>
+
+	<target name="cmap_legacysrc.zip" depends="init" unless="cmap_legacysrc.zip">
```

Comment:
hmm, we don't need to distribute zipped source code.  I think these steps can be removed.